### ISSUE TITLE
Form POST params missing in Chrome

### ISSUE THREAD

**Issue by kbeckman** [PolymerElements/paper-input#542]
### Description
When using the `2.0` version of the `paper-input` element with Polymer 2.0, form POST params are missing in Chrome. Form POST params work fine with the `paper-input` element using Firefox (Polymer 2.0) or when using Polymer 1.9 for both Firefox or Chrome.

### Expected Outcome
The following snippet should submit the `app_form[github_url]` form element using either Polymer 1.9 or 2.0 in both Chrome and Firefox.

```html
<form action="/en/apps" method="post">
  <paper-input label="Github URL"
               name="app_form[github_url]"
               value="http://github.com/rails">
  </paper-input>
  <input type="submit" value="submit-me">
</form>
```

_Form POST data from Rails web server log... This can also be verified from the Chrome/Firefox developer tools (network tab)... In the output below, the `locale` parameter can be ignored. This is a parameter generated by a parameter in our URL and added to the parameters hash by the Rails server._
```
Started POST "/en/apps" for ::1 at 2017-06-04 21:35:04 -0500
Processing by AppsController#create as HTML
  Parameters: {"app_form"=>{"github_url"=>"http://github.com/rails"}, {"locale"=>"en"}}
app_form[github_url]:http://github.com/rails
```

#### With Firefox (param present)...
```
Started POST "/en/apps" for 127.0.0.1 at 2017-06-04 22:01:18 -0500
Processing by AppsController#create as HTML
  Parameters: {"app_form"=>{"github_url"=>"http://github.com/rails"}, {"locale"=>"en"}}
Redirected to http://localhost:3000/en/apps/new
Completed 302 Found in 0ms
```

#### With Chrome (param missing)...
```
Started POST "/en/apps" for 127.0.0.1 at 2017-06-04 22:00:12 -0500
Processing by AppsController#create as HTML
  Parameters: {"locale"=>"en"}
Redirected to http://localhost:3000/en/apps/new
Completed 302 Found in 0ms
```
### Steps to reproduce

1. Put a `paper-input` element in the page inside a form element with `action="post".
2. Add an `<input type="submit">` element to trigger the POST.
3. Click the submit button.
4. Observe missing POST params with Polymer 2.0 / Chrome.

### Browsers Affected

- [x] Chrome
- [ ] Firefox
- [ ] Safari 9
- [ ] Safari 8
- [ ] Safari 7
- [ ] Edge
- [ ] IE 11
- [ ] IE 10

**Comment by notwaldorf** [comment]
In updating from Polymer 1 to 2, you also have shadow DOM enabled, and `<paper-input>` in a `<form>` element with shadow DOM enabled will not work -- native forms only work with the native `<input>` element. If you want to submit a `paper-input` you need to either use `iron-form`, or manually copy its value to a hidden input, like this:
```
<form>
  <paper-input value="{{a-value}}"></paper-input>
  <input type="hidden" value="[[a-value]]" name="name-for-submission>
</form>
```

**Comment by kbeckman** [comment]
Thanks for the feedback, @notwaldorf. We took your advice and got our form(s) working using Bower to install the `iron-form#2.0-preview` from Github. We'll change that up when `iron-form` is actually released at `2.0`. Thanks for the help!
